net/src/ipc_net_worker.rs

```diff
@@ -146,38 +160,41 @@ impl IpcNetWorker {
                 Ok(out)
             }),
             None,
+            endpoint,
         )
     }
 
     // -- private -- //
 
-    /// create a new IpcNetWorker instance pointing to a process that we spawn here
-    fn priv_spawn(
+    /// Constructor with IpcNetWorker instance pointing to a process that we spawn here
+    fn priv_new_with_spawn(
```

Comment:
fyi for merge conflicts: https://github.com/holochain/holochain-rust/pull/783/files#diff-ab813df663015fab8c20ce6f83b117c7